src/components/TopBar/createMenu.jsx

```diff
@@ -69,7 +70,9 @@ export default function createMenu({
       const menu = isOpen ?
         (
           <div
-            className="top-bar__menu"
+            className={classnames('top-bar__menu',
+              menuClass,
+            )}
```

Comment:
Not a huge deal but I feel like this could all go on one line?

Reply:
Done.